kubernetes/uninstall.sh

```diff
@@ -6,7 +6,13 @@ uninstall_airflow() {
 
   # Remove port forwarding
   PORT_FORWARD_PROCESS=`ps -ef | grep "kubectl port-forward airflow" | grep -v "grep" | awk '{ print $2 }'`
-  kill ${PORT_FORWARD_PROCESS}
+  if [ -z "$PORT_FORWARD_PROCESS" ]
```

Comment:
I like putting the if/then on one line with something like:
``` if [ whatever ]; then ```
It makes the code a little cleaner in my mind.